src/main.rs

```diff
@@ -167,8 +176,25 @@ impl Args {
             .value_of_lossy("votes")
             .unwrap()
             .parse()?;
+        let (rename_action, dest_dir): (RenameAction, Option<PathBuf>) =
+            if matches.is_present("symlink") {
+                if cfg!(windows) {
+                    failure::bail!(
+                        "symlink not supported on Windows, try hardlink");
+                }
+                (RenameAction::Symlink, Some(matches.value_of_os("symlink")
+                                             .unwrap()
+                                             .to_os_string().into()))
```

Comment:
Could you please format the code in a style consistent with the surrounding code? e.g.,

```
(RenameAction::Symlink, Some(
    matches.value_of_os("symlink")
        .unwrap().to_os_string().into()
))
```

Reply:
of course!